ios/ElectrodeApiImpl/APIImpls/Core/ENCoreDelegate.swift

```diff
@@ -26,7 +26,10 @@ import UIKit
         if let miniAppVC = viewController as? MiniAppNavViewController {
             self.viewIdentifier = UUID().uuidString
             let viewIdProperty = [ENCoreDelegate.KEY_UNIQUE_VIEW_IDENTIFIER: viewIdentifier] as [AnyHashable : Any]
-            self.createView(name: miniAppVC.miniAppName, properties: combineRouteData(dictionary1: miniAppVC.properties, dictionary2: viewIdProperty))
+            let extraProperties = miniAppVC.pushToExistingViewController ?
+                combineRouteData(dictionary1: viewIdProperty, dictionary2: miniAppVC.globalProperties) :
```

Comment:
Why do we need to add this to the core delegate? globalProperties is only applicable to those who use navigate, right ?

Reply:
we'll need this if we push a MiniappNavViewController on top of a existing NavigationController